website/docs/d/synthetics_private_location.html.markdown

```diff
@@ -23,6 +23,18 @@ resource "newrelic_synthetics_monitor" "foo" {
 }
 ```
 
+-> This data source doesn't work for `scripted_api`, `scripted_browser` and `step` monitors which works with the latest script runtime.
```

Comment:
This data source is not compatible with the `scripted_api`, `scripted_browser` and `step` monitors because private minion does not yet support the latest synthetics runtime.